fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatAuthenticationConfig.java

```diff
@@ -98,40 +96,16 @@ FiatWebSecurityConfigurerAdapter fiatSecurityConfig(FiatStatus fiatStatus) {
     private final FiatStatus fiatStatus;
 
     private FiatWebSecurityConfigurerAdapter(FiatStatus fiatStatus) {
+      super(true);
```

Comment:
this disables defaults, our auth filter chain for fiat is mostly about having the FiatAuthenticationFilter in there and falling back to anonymous